tests/client_auth_revocation.rs

```diff
@@ -14,8 +14,6 @@
 
 extern crate webpki;
 
-use webpki::EndEntityOrCa;
```

Comment:
This is not actually from clippy, but from the compiler lints. Maybe add some gunk to the CI workflow to make sure it denies warnings?

Reply:
Ah good catch. I'll look at that in a follow-up.

Reply:
https://github.com/rustls/webpki/pull/86